website/blog/2021-02-22-7.13.0.md

```diff
@@ -0,0 +1,255 @@
+
+Additionally, we added support for some new Flow and TypeScript features.
+
+You can read the whole changelog [on GitHub](https://github.com/babel/babel/releases/tag/v7.12.0).
+
+<!-- truncate -->
+
+### Funding Update
+
+We have joined the "GitHub Sponsors for organizations" program, so you can now sponsor us on GitHub 😊.
+
+<iframe src="https://github.com/sponsors/babel/button" title="Sponsor Babel on GitHub" height="35" width="116" style="border: 0; margin: auto; display: block"></iframe>
+
+These funds are used to pay one full-time and three part-time maintainers, to sustain the project developement, and ensure long-term improvements both regarding stability and new features.
+
+All of our donations and expenses are transparent on our [Open Collective](https://opencollective.com/babel) page, where we will also track donations coming from GitHub Sponsors.
+
+We welcome donations both from individual and from companies: if your company is interested in becoming a Gold tier sponsor ($1k/month) and wants to discuss more please reach out to [team@babeljs.io](mailto:team@babeljs.io)!
+
+## Highlights
+
+### Top-level `targets` option ([#12189](https://github.com/babel/babel/pull/12189), [RFC](https://github.com/babel/rfcs/pull/2))
+
+`targets` is the option that made `@babel/preset-env` the most powerful preset we have ever had: it can automatically choose what syntax to transform and what polyfills to inject based on your target browsers.
+
+However, other plugins can also benefit from knowing your targets. For example, you currently have to specify the targets twice if you are using our [new polyfill plugins](https://github.com/babel/babel-polyfills). With the new options you can lift it up to the top-level, specifying it only once:
+
+<table>
+<tr>
+<th>Old configuration</th>
+<th>New configuration</th>
+</tr>
+<tr>
+<td>
+
+```json
+{
+  "presets": [
+    ["@babel/preset-env", {
+      "targets": ">1%, not ie 11"
+    }]
+  ],
+  "plugins": [
+    ["polyfill-es-shims", {
+      "targets": ">1%, not ie 11"
+    }]
+  ]
+}
+```
+
+</td>
+<td>
+
+```json
+{
+  "targets": ">1%, not ie 11",
+  "presets": ["@babel/preset-env"],
+  "plugins": ["polyfill-es-shims"]
+}
+
+
+
+
+
+
+
+```
+
+</td>
+</tr>
+</table>
+
+We recommended converting configs to use the top-level `targets` option since:
+
+1. It better handles the [`esmodules: true`](TODO: Link) target. (it's intersected with the other targets, rather than replacing them)
+2. We are exploring how to use it in more plugins to generate more optimized output
+
+You can read the details about this new option in its [RFC](TODO: Link).
+
+> 🔮 In the future we might explore moving `@babel/preset-env` into `@babel/core` so that you don't have to install one additional package to get started with Babel. This new option can be seen as the first step in that direction!
```

Comment:
```suggestion
> 🔮 In the future we might explore moving `@babel/preset-env` into `@babel/core` so that you don't have to install an additional package to get started with Babel. This new option can be seen as the first step in that direction!
```